modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ContentsSectionDisplayContext.java

```diff
@@ -27,15 +29,18 @@ public class ContentsSectionDisplayContext extends BaseSectionDisplayContext {
 	public ContentsSectionDisplayContext(
 		DepotEntryLocalService depotEntryLocalService,
 		GroupLocalService groupLocalService,
-		HttpServletRequest httpServletRequest, Language language,
-		ObjectDefinitionService objectDefinitionService,
+		HttpServletRequest httpServletRequest,
+		ModelResourcePermission<ObjectEntryFolder>
+			objectEntryFolderModelResourcePermission,
+		Language language, ObjectDefinitionService objectDefinitionService,
```

Comment:
Constructor params shoyld be alphabetical ordered

HttpServletRequest, Language, ModelResourcePermission, ObjectDefinitionService

Reply:
done!